### DISCUSSION TITLE
moisture fraction

### DISCUSSION THREAD

**Discussion by SaharRasoulpour** [firemodels/fds#13532] [Q&A] [answered]
Hello, When I try to use a moisture fraction of 0.7 with six different components in my case, I encounter an error: ERROR(303): MOISTURE_FRACTION on SURF Pine Needles exceeds the theoretical limit. I have seen examples in FDS where the moisture fraction is greater than 1 with two different components, but in my case, I am facing this issue. How can I resolve this problem?
&SURF ID                               = 'Pine Needles'
      COLOR                              = 'RED'
      MOISTURE_FRACTION       = 0.7
      SURFACE_VOLUME_RATIO = 8366.
      LENGTH                     = 0.07
      GEOMETRY                = 'CYLINDRICAL' 
      MATL_ID(1,1)              = 'LIPID'
      MATL_ID(1,2)              = 'DIGEST'
      MATL_ID(1,3)              = 'HEMI_CELLULOSE'
      MATL_ID(1,4)              = 'GLUCAN'
      MATL_ID(1,5)              = 'PHENOLIC'
      MATL_ID(1,6)              = 'ASH'
      MATL_MASS_FRACTION(1,1)   = 0.2273
      MATL_MASS_FRACTION(1,2)   = 0.0990
      MATL_MASS_FRACTION(1,3)   = 0.1744 
      MATL_MASS_FRACTION(1,4)   = 0.1809
      MATL_MASS_FRACTION(1,5)   = 0.0888
      MATL_MASS_FRACTION(1,6)   = 0.2296 /

**Comment by mcgratta** [comment]
This basically means that there is not enough pore volume to accommodate all that water. There should be more explanation in the User's Guide. If not, let me know.

**Reply by SaharRasoulpour** [reply]
When I normalized the apparent density of each component, such as reducing lignin's density from 1000 kg/m³ to 323.206 kg/m³, I ran my simulation using these new normalized values. However, I'm unsure if using the normalized values is correct. I reviewed the user guide but couldn’t find more information on this.

**Comment by mcgratta** [comment]
![image](https://github.com/user-attachments/assets/d0a4cefb-f940-40c6-9aa6-f34e8df7828c)
![image](https://github.com/user-attachments/assets/09e1575f-3ab9-4891-88bc-ae1d9e7f73f8)

Basically, the denominator of this expression for the modified density goes to zero when you add too much water. FDS does not change the size of the particle when water is added, which means that there is a limit to how much you can add. If you want to include more moisture, make the particle bigger.

**Reply by SaharRasoulpour** [reply]
Thank you for the clarification. but, when I increase the particle size, I still encounter the same error. 
Additionally, I have another question: if I consider a moisture content of 70%, which is distributed over the surface of each pine needle (comprising six different components), can I assume that the moisture is uniformly divided among these components? In that case, instead of using a moisture fraction of 0.7, should I use 0.116 for each component?

**Reply by mcgratta** [reply, marked as answer]
We added `MOISTURE_FRACTION` as a convenience to users in the wildland fire community who are accustomed to it. I suggest that you not use `MOISTURE_FRACTION` but instead just add your moisture as another material component. Then the typical rules will apply to your water and your vegetation. It will all be mixed together.
